docs/src/xhtml/components/tags/gallery.xhtml

```diff
@@ -190,28 +162,18 @@
 			</figure>
 		</section>
 		<section>
-			<h3>List of Tags</h3>
+			<h3>List of tags</h3>
```

Comment:
This was capitalized on purpose, `Tag` is the name of the component. We could use `<code />` or something to make it obvious.

Reply:
Ah, ok. Understood.